daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/AlphaEquiv.scala

```diff
@@ -7,7 +7,7 @@ import com.daml.lf.language.Ast._
 
 import scala.annotation.tailrec
 
-private[validation] object AlphaEquiv {
```

Comment:
by putting the Updgradability in validation package  check, we can avoid that.

Reply:
Fixed